src/main/java/com/reservation/conference/service/CorpService.java

```diff
@@ -0,0 +1,40 @@
+package com.reservation.conference.service;
+
+import com.reservation.conference.dto.CorpJoinDto;
+import com.reservation.conference.mapper.CorpMapper;
+import com.reservation.conference.utils.SecurityUtil;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor    //초기화 되지 않는 final필드에 생성자 생성
+public class CorpService {
+
+    private final CorpMapper corpMapper;
+
+
+    public boolean join(CorpJoinDto corpJoinDto) throws Exception {
+        //아이디 중복 체크
+        boolean checkResult = checkCorpIdExist(corpJoinDto.getId());
+        if(checkResult) {
+            return false;
+        }else {
```

Comment:
큰 if else block 보다는 
```suggestion
        if(checkResult) {
            return false;
        }
        ... 그 다음 코드
```

처럼 사용을 고민해보시는건 어떨까요? `java Return Early Pattern` 검색 하시고 몇몇 글을 읽어보세요.

Reply:
1. 예상되는 긍정 결과는 함수의 끝에서 빨리 찾을 수 있다.
2. 이런 사고 흐름으로 구현하면 에러를 먼저 잡고 그 다음 불필요한 버그를 피하면서 안전하게 비지니스 로직을 구현할 수 있게 된다.
3. 함수가 에러 발생시 즉시 종료되기 때문에 의도되지 않은 코드가 실행될 가능성을 피할 수 있다.
의 등이 이유로 else block은 해당 코드에서 제거하는게 더 좋은 것 같습니다!

참고 : https://libertegrace.tistory.com/entry/Coding-Style-Return-Early-Pattern